eventapi.py

```diff
@@ -0,0 +1,56 @@
+import json
+from datetime import datetime
+from flask import request
+from flask_restful import Resource, abort
+
+from event import Event
+from eventvalidations import *
+
+def read_args(request):
+    return json.loads(request.data.decode('utf-8'))
+
+class EventAPI(Resource):
+    def get(self, e_id):
+        event = Event.get_by_id(e_id)
+        return vars(event)
+
+    def put(self, e_id):
+        event = Event.get_by_id(e_id)
+        data = read_args(request)
+        try:
+            if 'from_time' in data:
+                read_time(data['from_time'])
```

Comment:
lines 21 - 29 can all be reduced to 2 lines ... ill let you figure out how

Reply:
not sure what you were expecting